stage-prep/05-user/02-run-chroot.sh

```diff
@@ -6,7 +6,7 @@ pip3 install -U pip
 pip3 install wheel
 
 git clone https://github.com/OpenVoiceOS/ovos-docs-viewer
-mv ovos-docs-viewer/ovos_docs.py /home/ovos/.local/bin/ovos_docs.py
+mv ovos-docs-viewer/ovos_docs_viewer/ovos_docs.py /home/ovos/.local/bin/ovos_docs.py
```

Comment:
This whole section will need changed.  There is now a repo that this can be pip installed from.
https://github.com/OpenVoiceOS/ovos-docs-viewer

Reply:
I missed it, added the installation using PIP